Dart/maths/SumOfDigits.dart

```diff
@@ -0,0 +1,33 @@
+import 'dart:io';
+
+// Find sum of digits of number 
+int sum_digits(int number) {
+  int sum = 0;
+
+  // Extract individual digits and add to sum
+  while (number > 0) {
+    int digit = number % 10;
+    number ~/= 10;
+    sum += digit;
+  }
+  return sum;
+}
+
+// Main Function, with driver code
+void main() {
+  print("Enter a number:");
+  int input = int.parse(stdin.readLineSync()!);
+
+  // Call sum of digits function
+  int sum_of_digits = sum_digits(input);
+  print("Sum of digits in $input is $sum_of_digits");
+}
+/**
+Enter a number:
```

Comment:
YOu need to add the time complexity of the code before sample I/O. Rest seems fine.